config/_default/params/params.zh-cn.yaml

```diff
@@ -48,31 +65,48 @@ portraitPhotoWidths:
   - 600
   - 500
 lqipWidth: 20x
+smallLimit: '300'
 
-# footer/left side
+# TODO: Maybe mention Hosting? NetworkRedux? Github Pages? Cloudflare Pages?
+# Footer
 footer: >-
-  Powered by <a href="https://www.networkredux.com/">NetworkRedux</a>, <a
-  href="https://gohugo.io/">Hugo</a>, and <a
-  href="https://getdoks.org/">Doks</a>
+  Powered by <a class="text-muted" href="https://gohugo.io/">Hugo</a> and <a
+  class="text-muted" href="https://getdoks.org/">Doks</a>
```

Comment:
```suggestion
  class="text-muted" href="https://getdoks.org/">Doks</a>. <br/>
  Copyright (c) AegiSite <a href="https://github.com/Aegisub/aegisite/graphs/contributors">contributors</a>
    and <a href="https://github.com/computerfan/Chinese-aegisite/graphs/contributors">translators</a>.
```
![image](https://user-images.githubusercontent.com/5158738/159029117-e92d99b6-df31-40dd-a9dc-78d4cb646452.png)

Reply:
I like it. I'll add the same text without the "and translators" to the English version.